src/components/NavigationBar/style.css

```diff
@@ -0,0 +1,42 @@
+@import (less) "../Theme/constants.css";
+
+.navigation {
+  float: left;
+  position: absolute;
+  top: 16px;
+  margin-left: 16px;
+  text-align: left;
+  @media (--small-viewport) {
+    margin-left: 12px;
+  }
+}
+
+.back {
+  width: 72px;
+  height: 32px;
+  color: #2C3E4F;
+  font-size: 14px;
+  line-height: 17px;
+  border: none;
+  background-color: transparent;
+  cursor: pointer;
+}
+
+.iconBack {
```

Comment:
Minor: css tens to go from broader to more specific. so in order to follow this idea, this selector would be better named `backIcon`